### PR TITLE
AAP-18084 adds collection count variable to ansible automation hub variables table in appendix

### PR DESCRIPTION
**Pull request by hherbly**
See [AAP-18084](https://issues.redhat.com/browse/AAP-18084) for more detail.

### CODE REVIEW

**Review by simaishi — APPROVED** [review]
Looks good, thanks!

File: downstream/modules/platform/ref-hub-variables.adoc [hunk 1]
```diff
@@ -39,6 +39,11 @@ Default = `false`.
 You can also set `automationhub_backup_collections = false` and the backup/restore process does not then backup or restore `/var/lib/pulp`.
 
 Default = `true`
+| *`automationhub_collection_download_count`* | _Optional_
+
+Determines whether download count is displayed in the UI.
```
Comment (sayjadha) [inline downstream/modules/platform/ref-hub-variables.adoc:44]: 'in' the UI -> 'on' the UI.

**Review by sayjadha — COMMENTED** [review]
A minor comment, rest LGTM.

**Review by sayjadha — COMMENTED** [review]
Change LGTM.

**Pull request merged**